packages/tools/examples/videoTools/index.ts

```diff
@@ -60,106 +95,285 @@ rangeElement.oninput = () => {
+      bindings: [
+        {
+          mouseButton: MouseBindings.Primary, // Left Click
+        },
+      ],
+    });
+
+    // Set the old tool passive
+    toolGroup.setToolPassive(selectedToolName);
+
+    selectedToolName = <string>newSelectedToolName;
+  },
+});
+
+const nextButton = addButtonToToolbar({
+  id: 'Next',
+  title: '<',
   onClick() {
-    viewport.scroll(-1);
+    selectNextAnnotation(1);
   },
 });
 
+nextButton.parentElement.appendChild(document.createTextNode('Annotation'));
+
 addButtonToToolbar({
-  id: 'next',
-  title: 'next',
+  id: 'Previous',
+  title: '>',
   onClick() {
-    viewport.scroll(1);
+    selectNextAnnotation(-1);
   },
 });
 
-addButtonToToolbar({
-  id: 'jump',
-  title: 'jump to 50',
+let activeGroup = group1;
+
+const groupButton = addButtonToToolbar({
+  id: 'Group',
+  title: 'Group 1',
   onClick() {
-    viewport.setTime(50);
+    activeGroup.setVisible(false, baseEventDetail);
+    activeGroup.setActive(false);
+    activeGroup = activeGroup === group1 ? group2 : group1;
+    groupButton.innerText = activeGroup === group1 ? 'Group 1' : 'Group 2';
+    activeGroup.setVisible(true, baseEventDetail);
+    activeGroup.setActive(true);
   },
 });
 
-const playbackSpeeds = [
-  '0',
-  '0.075',
-  '0.15',
-  '0.25',
-  '0.5',
-  '0.75',
-  '1',
-  '2',
-  '3',
-  '4',
-  '10',
-];
+function togglePlay(toggle = undefined) {
+  if (toggle === undefined) {
+    toggle = viewport.togglePlayPause();
+  } else if (toggle === true) {
+    viewport.play();
+  } else {
+    viewport.pause();
+  }
+  playButton.innerText = toggle ? 'Play' : 'Pause';
+}
 
-const toolbar = document.getElementById('demo-toolbar');
-const rateTitle = document.createElement('div');
-rateTitle.style.display = 'inline';
-rateTitle.innerText = 'Playback Rate:';
-toolbar.appendChild(rateTitle);
-addDropdownToToolbar({
-  options: { values: playbackSpeeds, defaultValue: '1', id: 'frameRate' },
-  onSelectedValueChange: (newSelectedToolNameAsStringOrNumber) => {
-    const newPlaybackSpeed = Number(newSelectedToolNameAsStringOrNumber);
-    viewport.setPlaybackRate(newPlaybackSpeed);
+addButtonToToolbar({
+  id: 'Set Range [',
+  title: '[',
+  onClick() {
+    const annotation = getActiveAnnotation();
+    if (annotation) {
+      const rangeSelection =
+        annotationFrameRange.annotationToFrameRange(annotation);
+      const frame = viewport.getFrameNumber();
+      const range = Array.isArray(rangeSelection)
+        ? rangeSelection
+        : [rangeSelection, viewport.numberOfFrames];
+      range[0] = frame;
+      range[1] = Math.max(frame, range[1]);
+      annotationFrameRange.setFrameRange(
+        annotation,
+        range as [number, number],
+        baseEventDetail
+      );
+      viewport.setFrameRange(range);
+      viewport.render();
+    }
   },
 });
 
-const scrollSpeeds = ['1 f', '2 f', '4 f', '0.5 s', '1 s', '2 s', '4 s'];
+addButtonToToolbar({
+  id: 'Set Current',
+  title: 'Single Image',
+  onClick() {
```

Comment:
can you explain in the description that this means for navigation back and forth it is single and not a range